docker/Dockerfile

```diff
@@ -93,6 +98,8 @@ FROM debian:buster-slim
 RUN apt-get update && apt-get install -y locales-all && rm -rf /var/lib/apt/lists/*; \
     mkdir -p config && mkdir -p data && mkdir -p app
 
+COPY --from=builder /usr/local/lib /usr/local/lib
+COPY --from=builder /etc/ld.so.conf.d/mylibs.conf  /etc/ld.so.conf.d/mylibs.conf
```

Comment:
```
COPY --from=builder /usr/local/lib /usr/local/lib <- this is necessary
COPY --from=builder /etc/ld.so.conf.d/mylibs.conf  /etc/ld.so.conf.d/mylibs.conf <- this can probs be removed because I do it in `entrypoint.sh` now
```